test/test_vector.c

```diff
@@ -0,0 +1,119 @@
+/* Deterministic randombytes by Daniel J. Bernstein */
```

Comment:
We'll need to figure out the license for this one. Like this it cannot be merged.

Reply:
This test is not important. I'll take it out.

Reply:
removed.